.gitignore

```diff
@@ -133,3 +133,30 @@ dmypy.json
 
 scripts/minio-conf/
 scripts/minio-data/
+
+## Pycharm ##
+
+.idea/
+.idea_modules/
+*.iml
+modules.xml
+*.ipr
+
+### VisualStudioCode ###
+.vscode/
+!.vscode/settings.json
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
+*.code-workspace
+
+# Local History for Visual Studio Code
+.history/
+
+### VisualStudioCode Patch ###
+# Ignore all local history of files
+.history
```

Comment:
@saltgen - does that mean both directory and file are present?

Reply:
Still not sure if VS Code creates both `.history` folder and file. 

Regardless, tested locally by creating `.history`(non-empty) folder and another `.history` file. 

`.history` entry on .gitignore takes care of both, changes pushed :+1: